README.md

```diff
@@ -4,90 +4,109 @@ FHIR Converter is an open source project that enables conversion of health data
 
-The Converter makes use of templates that define the mappings between different data formats.
-The templates are written in [Liquid](https://shopify.github.io/liquid/) templating language and make use of custom [filters](docs/FiltersSummary.md), which make it easy with work with HL7 v2 messages.
+The converter comes with a few ready-to-use templates. If needed, you can create a new template, or modify existing templates to meet your specific conversion requirements.
 
-The converter comes with a few ready to use templates for HL7v2 to FHIR conversion. These templates are based on the [spreadsheet](https://docs.google.com/spreadsheets/d/1PaFYPSSq4oplTvw_4OgOn6h2Bs_CMvCAU9CqC4tPBgk/edit#gid=0) created by the HL7 [2-To-FHIR project](https://confluence.hl7.org/display/OO/2-To-FHIR+Project). If needed, you can create new, or modify existing templates to meet your specific conversion requirements.
+FHIR Converter with DotLiquid engine transforms the input data into FHIR bundles that are persisted to a FHIR server. The converter is integrated into both [Azure API for FHIR](https://azure.microsoft.com/en-us/services/azure-api-for-fhir/) and [FHIR Server](https://github.com/microsoft/fhir-server) as a [$convert-data](https://docs.microsoft.com/en-us/azure/healthcare-apis/convert-data) operation. It is also available as a command line tool.
 
-FHIR Converter with DotLiquid engine is integrated into the [Azure API for FHIR](https://azure.microsoft.com/en-us/services/azure-api-for-fhir/), and [FHIR Server for Azure](https://github.com/microsoft/fhir-server) as the [$convert-data](https://docs.microsoft.com/en-us/azure/healthcare-apis/convert-data) operation. In addition, it is also available as a command-line tool. The converter transforms the input data into FHIR bundles that can be persisted to a FHIR server.
+## Using $convert-data
 
-This project consists of the following components:
+The `$convert-data` operation is integrated into both Azure API for FHIR and FHIR server to run as part of the service. After enabling `$convert-data` in your server, you can make API calls in the form of ```<<your FHIR server URL>>/$convert-data``` to the server to convert your data into FHIR.
 
-1. A command-line tool) for converting data and managing templates.
-2. [Templates](data/Templates) for HL7 v2 to FHIR conversion.
-3. [Sample data](data/SampleData) for testing purpose.
+In the API call request body, you would include the following parameters:
 
-## Using the FHIR Converter
+| Parameter Name      | Description | Accepted values |
+| ----------- | ----------- | ----------- |
+| inputData      | Data to be converted. | A valid JSON String|
```

Comment:
The `inputData` isn't a json string. It's just a string for `HL7v2`, and a xml for `C-CDA`.

Reply:
Since Boya suggested to remove this convert-data section as it is already on FHIR server and Azure API for FHIR docs, I will mark this as resolved. I have updated these changes directly on the FHIR server doc: https://github.com/microsoft/fhir-server/pull/2164